code/modules/power/cell.dm

```diff
@@ -204,14 +204,23 @@
 	materials = list(MAT_GLASS=60)
 	rating = 3
 	chargerate = 1500
+	
+/obj/item/weapon/stock_parts/cell/guncrafthigh  //Battery variants for guncrafting
+	name = "high-capacity gun power cell"
+	origin_tech = "powerstorage=2"
+	icon_state = "hcell"
+	maxcharge = 10000

```

Comment:
No. The charge is STILL to high on this cell, and all the other guncrafting cells. This is just re-adding the original problem which goof had in, just slightly nerfed, since the high frame  starts off already with 10x the shots than the base frame, with the biggest downgrade being the bluespace frame going from 40x to 20x, which would probably still be about 200+ shots. I recommend the following.

High: 1500
Super: 2000
Hyper: 2500
Bluespace: 4000(To make it actually be a decent upgrade and worth trying to get)

Also, instead of making entirely new paths for this, such as '/obj/item/weapon/stock_parts/cell/guncrafthigh', why not just make that powercell for example a child of '/obj/item/weapon/stock_parts/cell/high'

For example:
/obj/item/weapon/stock_parts/cell/high/guncraft
	name = "high capacity gun power cell
	maxcharge = 1500
	chargerate = 1500